threads/test_thread_rand.c

```diff
@@ -0,0 +1,88 @@
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/threads.h>
+
+#include "unity_fixture.h"
+
+
+typedef struct {
+	uint16_t time;
+	uint16_t id;
+} threads_self_t;
+
+
+static int cthread = 10, fin = 0;
+
+
+static char stack[10 * 512];
```

Comment:
We agreed to use common structures holding common. global attributes attributes. For code clarity is better to use two dimensional array.

Reply:
Two dimensional array implemented. `thread_common` structure also created.